apps/web/README.md

```
@@ -1,128 +1,64 @@
+**This app needs no `.env` of its own.** `VITE_BACKEND_URL` is deliberately
+empty so API calls stay relative; setting it is only necessary when the API
+really is on another origin, and note that it is inlined at build time.
 
-Direct Supabase auth integration:
+## Layout
 
-```typescript
-import { getCurrentUser, signOut } from '../lib/supabase';
-
-const user = await getCurrentUser();
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)